docs/source/guides/user/chapters/configuring.rst

```diff
@@ -216,6 +223,8 @@ Example using the configuration file:
   [core]
   verbose = 'true'
 
+As explained at the "strings" section, here you can also ommit the quotes.
```

Comment:
I think a more natural example is to simply use the `verbose = true` without the quotes in the example above.

Reply:
Agree.